### ISSUE TITLE
Proxmark easy support? 

### ISSUE THREAD

**Issue by Dutchy-79** [RfidResearchGroup/RFIDtools#77]
Hi,

There's no question section here but is there a way to use proxmark easy with this?

Ty

**Comment by UltrashRicco** [comment]
Yes, Proxmark easy works with RFIDtools, however, you will have to compile and flash release v4.13441 of the Prowmark3 firmware to ensure compatibility: https://github.com/RfidResearchGroup/proxmark3/releases/tag/v4.13441

At least, that is what works for me with a Proxmark easy with only 256kB of memory (mond the compilations options so as not to go over that limit, if your device only has 256kB of memory).

RFIDtools is somewhat behind the latest Proxmark3 client and features, and I see no active development being done. I would like to contribute but I have little experience myself!

Great APP for on-the-go use of the Proxmark3 with no computer, all just from the smartphone!
